pkg/debug/scripts/collect-info.sh

```diff
@@ -433,9 +443,17 @@ find /sys/kernel/security -name "tpm*" | while read -r TPM; do
     fi
 done
 
+if [ -n "$READ_LOGS_DAYS" ]; then
+    mkdir -p "$LOG_TMP_DIR"
+    # Find and copy log files from /persist/newlog to $LOG_TMP_DIR in previous days
+    find /persist/newlog -type f -mtime -"$READ_LOGS_DAYS" -exec cp -p {} "$LOG_TMP_DIR" \;
```

Comment:
The reason to create a link of /persist/newlog it's to avoid coping log files that can be huge, but here you are copying anyways, which can lead the device to run out of disk space. This can be mitigated by creating symbolic links (inside $LOG_TMP_DIR) instead of copy all the files...

Reply:
good point Rene, will do. Although i'm only getting 10 days of logs and it's just few mega bytes here. but yes let me try to use 'ln'.